packages/ui/src/components/SelectInputV2/SelectBar.tsx

```diff
@@ -0,0 +1,388 @@
+import styled from '@emotion/styled'
+import { Icon } from '@ultraviolet/icons'
+import type { Dispatch, RefObject, SetStateAction } from 'react'
+import { useEffect, useMemo, useRef, useState } from 'react'
+import { Button } from '../Button'
+import { Stack } from '../Stack'
+import { Tag } from '../Tag'
+import { Text } from '../Text'
+import type { DataType, OptionType } from './types'
+import { INPUT_SIZE_HEIGHT, SIZES_TAG } from './types'
+
+type SelectBarProps = {
+  size: 'small' | 'medium' | 'large'
+  clearable: boolean
+  disabled: boolean
+  readOnly: boolean
+  value: OptionType | undefined
+  placeholder: string
+  multiselect: boolean
+  success?: string
+  error?: string
+  onChange?: (value: string[]) => void
+  autoFocus?: boolean
+  innerRef: RefObject<HTMLDivElement>
+  selectedValues: OptionType[]
+  setSelectedValues: Dispatch<SetStateAction<OptionType[]>>
+  isDropdownVisible: boolean
+  setIsDropdownVisible: Dispatch<SetStateAction<boolean>>
+  setAllSelected: Dispatch<SetStateAction<boolean>>
+  options: DataType
+  setSelectedGroups: Dispatch<SetStateAction<string[]>>
+}
+
+type StyledInputWrapperProps = {
+  size: 'small' | 'medium' | 'large'
+  isDropdownVisible: boolean
+  state: 'neutral' | 'danger' | 'success'
+  disabled: boolean
+  readOnly: boolean
+}
+
+type DisplayValuesProps = {
+  multiselect: boolean
+  refTag: RefObject<HTMLDivElement>
+  width?: number
+  nonOverflowedValues: OptionType[]
+  disabled: boolean
+  readOnly: boolean
+  selectedValues: OptionType[]
+  setSelectedValues: Dispatch<SetStateAction<OptionType[]>>
+  onChange?: (value: string[]) => void
+  overflowed: boolean
+  overflowAmount: number
+  setAllSelected: Dispatch<SetStateAction<boolean>>
+}
+const StateStack = styled(Stack)`
+  padding-right: ${({ theme }) => theme.space['2']};
+  right: 0px;
```

Comment:
since Stack is not positionned (absolute or fixed), this `right` has no effect no ? (if it does mb a comment could help)